app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    source, target = parts[1], parts[2]
```

Comment:
First, check if the length of `parts` is 3 and you can unpack command as well

Reply:
how i can do it with one "return"?